src/theme/TabItem/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import TabItem from '@theme-original/TabItem';
+import type TabItemType from '@theme/TabItem';
+import type {WrapperProps} from '@docusaurus/types';
+import styles from './styles.module.css';
+
+type Props = WrapperProps<typeof TabItemType>;
+
+// Wrapper for TabItem to hide the tab content visually but keep it in the DOM for scrapers
```

Comment:
I suggest adding a link to https://docusaurus.io/docs/swizzling#wrapping here. Since the swizzling process is implicit it could be hard to grasp to future readers, so documenting it a bit sounds like a good idea.